plugins/parsers/avro/parser_test.go

```diff
@@ -115,6 +115,7 @@ func BenchmarkParsing(b *testing.B) {
 
 	b.ResetTimer()
 	for n := 0; n < b.N; n++ {
+		//nolint:errcheck // Benchmarking so skip the error check to avoid the unnecessary operations
 		_, _ = plugin.Parse(benchmarkData)
```

Comment:
Please remove the underscores in this case
```suggestion
		plugin.Parse(benchmarkData)
```
Same for the instances below.